sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/Program.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+
+namespace Kmd.Logic.Audit.Client.SampleLargeAuditEvents
+{
+    public static class Program
+    {
+        public static void Main(string[] args)
+        {
+            var config = new ConfigurationBuilder()
+                .AddCommandLine(args)
+
+                // Without *some* values in the config, we get a null instance of `ProgramConfig`
+                // however if we ensure at least one value is in the config collection, then we get
+                // an instance, no matter what the key/value is.
+                .AddInMemoryCollection(initialData: new Dictionary<string, string> { { "dummy", "value" }, })
+                .Build()
+                .Get<ProgramConfig>();
+
+            if (string.IsNullOrEmpty(config?.Ingestion?.ConnectionString))
+            {
+                Console.WriteLine("You need to specify a connection string (e.g. --Ingestion:ConnectionString=\"test\")");
+                return;
+            }
+
+            var clientConfig = new Kmd.Logic.Audit.Client.SerilogLargeAuditEvents.SerilogLargeAuditEventClientConfiguration
+            {
+                EventSource = config.Ingestion.EventSource ?? $"{typeof(Program).Assembly.GetName().Name} on {Environment.MachineName}",
+                AuditEventTopic = config.Ingestion.AuditEventTopic,
+                EventHubConnectionString = config.Ingestion.ConnectionString,
+                EnrichFromLogContext = config.Client.EnrichFromLogContext,
+                StorageAccountName = config.Ingestion.BlobAccountName,
+                StorageConnectionString = config.Ingestion.BlobConnectionString,
+                StorageContainerName = config.Ingestion.BlobContainerName
+            };
+
+            // Create data for audit event
```

Comment:
Those sample don't provide any value in terms of testing a new _client_. As an event data I would use some bigger payload (>256KB) to test both saving blob in storage and sending event.

Reply:
Increased loop count to increase payload to 2 MB